src/tests/mocks/reentrancy_mock.cairo

```diff
@@ -16,58 +17,59 @@ trait IReentrancyMock<TState> {
 
 #[starknet::contract]
 mod ReentrancyMock {
-    use openzeppelin::security::reentrancyguard::ReentrancyGuard;
+    use openzeppelin::security::reentrancyguard::ReentrancyGuard as reentrancy_guard_comp;
     use openzeppelin::tests::mocks::reentrancy_attacker_mock::IAttackerDispatcher;
     use openzeppelin::tests::mocks::reentrancy_attacker_mock::IAttackerDispatcherTrait;
     use starknet::ContractAddress;
     use starknet::get_contract_address;
     use super::IReentrancyGuardedDispatcher;
     use super::IReentrancyGuardedDispatcherTrait;
 
+    component!(path: reentrancy_guard_comp, storage: reentrancy_guard, event: ReentrancyGuardEvent);
+    impl InternalImpl = reentrancy_guard_comp::InternalImpl<ContractState>;
```

Comment:
```suggestion
    component!(path: reentrancy_guard_comp, storage: reentrancy_guard, event: ReentrancyGuardEvent);
    
    impl InternalImpl = reentrancy_guard_comp::InternalImpl<ContractState>;
```

I think it may be worth to separate this blocks even if there is just one impl.